hack/buildx.sh

```diff
@@ -0,0 +1,3 @@
+docker run --rm --privileged linuxkit/binfmt:v0.8
```

Comment:
Is this file used somewhere?

Reply:
This starts the emulator , we need to run this file before we start the multiarch build.

Reply:
Can you please add documentation at top of the script?

Reply:
Done